lib/partition_setup.pm

```diff
@@ -162,5 +162,15 @@ sub addlv {
     send_key $cmd{finish};
 }
 
+# On Xen PV "CDROM" is of the same type as a disk block device so YaST
+# naturally sees it as a "disk". We have to uncheck the "CDROM".
+sub unselect_xen_pv_cdrom {
+    if (check_var('VIRSH_VMM_TYPE', 'linux')) {
+        assert_screen 'select-hard-disk';
+        send_key 'alt-e';
```

Comment:
I see that previously it was 'alt-d' or alt-e key. Is it same now in both scenarios?

Reply:
Yes it is, see http://assam.suse.cz/tests/21#step/partitioning_filesystem/2 and http://assam.suse.cz/tests/22#step/partitioning_togglehome/1.